### PR TITLE
emit an error for invalid searches

### PR DESCRIPTION
**Pull request by armw4**
This basically gives `SearchPhaseExecutionException`s the opportunity to be properly emitted. I went out of my way to trigger this, but basically if you submit a key in your search that elasticsearch does not support, it will throw.

### Before (null reference)

screen shot 2015-10-14 at 12 41 47 am

### After (SearchPhaseExecutionException)

screen shot 2015-10-14 at 12 45 07 am

### CODE REVIEW

**Comment by armw4** [comment]
@lanetix/engineers

File: index.js [hunk 1]
```diff
@@ -42,11 +42,13 @@ export default class SearchStream extends Readable {
     this._boundNext = this._next.bind(this)
   }
 
-  _next (error, { hits: { hits, total }, _scroll_id }) {
+  _next (error, response) {
```
Comment (apechimp) [inline index.js]: Ahhh interesting...I guess destructuring has some drawbacks.

**Comment by apechimp** [comment]
:sparkles::sparkles::sparkles::sparkles:
:sparkles:LGTM:sparkles:
:sparkles::sparkles::sparkles::sparkles:

File: index.js (hunk 1 above)
Comment (armw4) [inline index.js]: The occasional gotcha but we shall "fight, fight, fight"

**Pull request merged**